src/components/RichEditor/index.jsx

```diff
@@ -88,9 +89,11 @@ class RichEditor extends React.Component {
             init={{
               branding: false,
               menubar: false,
-              plugins: 'legacyoutput link lists language',
+              selector: 'textarea',
+              plugins: 'legacyoutput link lists language directionality',
               statusbar: false,
-              toolbar: 'undo redo | bold italic underline | bullist numlist | link | language',
+              toolbar: 'undo redo | bold italic underline | bullist numlist | link | language | ltr rtl',
+              directionality: 'ltr',
```

Comment:
Can we pass the directionality as a prop to RichEditor from EditCourseForm directly? As it stands now there are two toggles: one for text fields and separate one for richEditor. I feel it would be better if toggling the button at top would toggle direction of RichText fields too

Reply:
+1